### PR TITLE
Add ValueSpecificationAction

### PR DESCRIPTION
**Pull request by marek-piirikivi**
### PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I have read, and I am following the [Contributor guide](https://github.com/gaphor/gaphor/blob/main/CONTRIBUTING.md)
- [x] I have read, and I understand the GNOME [Code of Conduct](https://wiki.gnome.org/Foundation/CodeOfConduct)

### PR Type
What kind of change does this PR introduce?

- [ ] Bug fix
- [x] Feature
- [ ] Chore (refactoring, formatting, local variables, other cleanup)
- [ ] Documentation content changes

### What is the current behavior?

Issue Number: [N/A](https://github.com/gaphor/gaphor/issues/2572)

### Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by marek-piirikivi** [comment]
@sz332 Please review if this is what you desired and also whether it is conforming with standards - I think I got it right. But I failed to locate where this stereotype "valueSpecification" originates from because I could not find it from UML not that I started checking it, so it might be SysML exclusive thing, but I am not entirely sure.

**Comment by sz332** [comment]
I checked in the standard and various tools, and I think you got it right. 

The <<valueSpecification>> is according to my understanding is not a stereotype but a visual aid for modelers to see that this action is actually a value specification action. I have seen this behaviour in some tools, but there are some that use just a label Value specification without the stereotype notation.

**Comment by sz332** [comment]
@danyeaw can you please review? From my point of view it is a good V1 feature, gets the job done for now.

**Review by amolenaar — APPROVED** [review]
Looks like a valuable improvement :wink: .

**Comment by marek-piirikivi** [comment]
> Looks like a valuable improvement.

@amolenaar Ah, I finally see what you did there! :joy: Nice one!

File: gaphor/ui/icons/hicolor/scalable/actions/gaphor-value-specification-action-symbolic.svg
Comment (danyeaw) [inline gaphor/ui/icons/hicolor/scalable/actions/gaphor-value-specification-action-symbolic.svg:1]: I think we need to add this to the `gaphor/ui/icons/stencil.svg` and the Makefile in the same folder.

Comment (marek-piirikivi) [reply gaphor/ui/icons/hicolor/scalable/actions/gaphor-value-specification-action-symbolic.svg:1]: Could you please elaborate more on what exactly you want me to do? I found stancil.svg and makefile. I am not sure how to proceed.

Comment (sz332) [reply gaphor/ui/icons/hicolor/scalable/actions/gaphor-value-specification-action-symbolic.svg:1]: I had the same issue with InterfaceBlock, check that ticket (am in a meeting now :D )

Comment (danyeaw) [reply gaphor/ui/icons/hicolor/scalable/actions/gaphor-value-specification-action-symbolic.svg:1]: The stencil.svg has all the icons in it. Create a new icon in the stencil as a new layer and name the label and ID the name for the icon. Then add that name to the Makefile and use it to generate the icon with `make all`. Maybe we should add this to our docs.

Comment (marek-piirikivi) [reply gaphor/ui/icons/hicolor/scalable/actions/gaphor-value-specification-action-symbolic.svg:1]: I did that. As a result, a lot of files got modified. :thinking: I suppose it is fine and a result of me just using the latest inkscape.

Comment (marek-piirikivi) [reply gaphor/ui/icons/hicolor/scalable/actions/gaphor-value-specification-action-symbolic.svg:1]: I see that call behavior action is also missing from the stencil. I'll add that as well.

**Review by danyeaw — APPROVED** [review]
Thanks @marek-piirikivi!

**Pull request merged**